core/gather/driver/wait-for-condition.js

```diff
@@ -446,6 +446,21 @@ async function waitForFullyLoaded(session, networkMonitor, options) {
   // CPU listener. Resolves when the CPU has been idle for cpuQuietThresholdMs after network idle.
   let resolveOnCPUIdle = waitForNothing();
 
+  if (log.isVerbose()) {
```

Comment:
Why is this `if` clause necessary? Since you're using `log.verbose` wouldn't these messages get filtered out under normal circumstances anyway?